### PR TITLE
Update groups-naming-policy.md

### PR DESCRIPTION
**Pull request by msbemba**
Added a note on what Special characters are allowed

### CODE REVIEW

**Comment by msbemba** [comment]
Issue#https://github.com/MicrosoftDocs/microsoft-365-docs/issues/1965

**Comment by mypil** [comment]
@JohanFreelancer9 - kindly do the copy editor review for this PR. Thanks.

File: microsoft-365/admin/create-groups/groups-naming-policy.md [hunk 1]
```diff
@@ -76,6 +76,9 @@ It's recommended that you use attributes that have values filled in for all user
 
 - Prefixes and suffixes can contain special characters supported in group name and group alias. When the prefixes and suffixes contain special characters that are not allowed in the group alias, they are only applied to the group name. So in this case, the prefixes and suffixes applied to group name would be different from the ones applied to the group alias.
 
+> [!NOTE]
+> A period (.) or a hyphen (-) is permitted anywhere in the Group name, except at the beginning or end of the name. An underscore (_) is permitted anywhere in the Group name. This includes at the beginning or end of the name.
```
Comment (JohanFreelancer9) [inline microsoft-365/admin/create-groups/groups-naming-policy.md]:
```suggestion
> A period (.) or a hyphen (-) is permitted anywhere in the group name, except at the beginning or end of the name. An underscore (_) is permitted anywhere in the group name, including at the beginning or end of the name.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Pull request merged**